kevm-pyk/src/tests/integration/test-data/examples/ERC20.sol

```diff
@@ -0,0 +1,94 @@
+// THIS IS A BUGGY ERC20
+// DO NOT USE FOR ANYTHING
+
+pragma solidity >=0.6.0;
+
+contract ERC20 {
```

Comment:
I had to copy these, `solc` refuses to compile contracts that are not in the tree of the current working directory.